libs/shared/src/config/trigger-workflow.ts

```diff
@@ -0,0 +1,20 @@
+export enum QueueNameEnum {
+  INBOUND_PARSE_MAIL = 'inbound-parse-mail',
+  METRICS = 'metric',
+  STANDARD = 'standard',
+  WEB_SOCKETS = 'ws_socket_queue',
+  WORKFLOW = 'trigger-handler',
```

Comment:
AS the trigger-handler lives inside the Workflow module made the decision for this key.